.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
Nit: Consider including a comment explaining why.

Reply:
Do you mean a comment in this PR? Or a comment in the source code? I updated the README to call this out, in case that's relevant.

Reply:
Sorry for the confusion. The README update about v10 is in a different PR that isn't merged yet.